### ISSUE TITLE
*.ics calender that was imported without problems in calendar < 2.0 can't be imported anymore

### ISSUE THREAD

**Issue by sreutter** [nextcloud/calendar#2031]
### Steps to reproduce
https://eb-mainz.de/wir-kommen-zu-ihnen/abfallkalender
generated a *.ics file with setting "MS Outlook, Google Kalender etc. mit Terminserien (nicht für Apple Kalender*) "
tried to import that ics file, doesn't work, no failure message at all.

### Expected behaviour
normal import of the ics file in an existing or new nextcloud calender

### Actual behaviour
Import fails

Start of the *.ics file:

BEGIN:VCALENDAR
PRODID:-//Grafik-Partner GmbH//Muellmax 7.0 MIMEDIR//EN
VERSION:2.0
METHOD:PUBLISH
BEGIN:VEVENT
DTSTART;VALUE=DATE:20200227
DTEND;VALUE=DATE:20200228
DESCRIPTION:Entsorgungsbetrieb der Stadt Mainz\n
	Zwerchallee 24\n
	55120 Mainz\n
	\n
	Telefon: +49 6131 12 34 56 (Abfallberatung)\n
	Telefax: +49 6131 12 34 83\n
	\n
	Der Entsorgungsbetrieb ist ein Eigenbetrieb der Stadt Mainz.
LOCATION:55118 Mainz-Neustadt, Wallaustraße 31
RRULE:FREQ=WEEKLY;UNTIL=20210103T000000Z;
 INTERVAL=1;BYDAY=TH;WKST=MO
RDATE;VALUE=DATE:
 20200228,
 20200408,
 20200417,
 20200429,
 20200522,
 20200605,
 20200612,
 20201222,
 20201230
EXDATE;VALUE=DATE:
 20200227,
 20200409,
 20200416,
 20200430,
 20200521,
 20200604,
 20200611,
 20201224,
 20201231
TRANSP:TRANSPARENT
SEQUENCE:0
UID:15828015991@muellmax.de
DTSTAMP:20200227T110639Z
SUMMARY:EB Restabfall-Behälter
CLASS:PUBLIC
CREATED:20200225T161546Z
LAST-MODIFIED:20200225T161546Z
URL:http://www.muellmax.de
END:VEVENT
END:VCALENDAR

## Server configuration detail

**Operating system:** Linux 4.4.0-173-generic #203-Ubuntu SMP Wed Jan 15 02:55:01 UTC 2020 x86_64

**Webserver:** Apache (cgi-fcgi)

**Database:** mysql 5.7.29

**PHP version:** 

7.3.8
Modules loaded: Core, date, libxml, openssl, pcre, sqlite3, zlib, bcmath, bz2, calendar, ctype, curl, dba, dom, hash, fileinfo, filter, ftp, gd, gettext, gmp, SPL, iconv, session, intl, json, ldap, mbstring, mysqli, pcntl, PDO, pdo_dblib, pdo_mysql, PDO_ODBC, pdo_pgsql, pdo_sqlite, pgsql, standard, posix, pspell, Reflection, imap, shmop, SimpleXML, snmp, soap, sockets, sodium, Phar, exif, sysvmsg, sysvsem, sysvshm, tidy, tokenizer, wddx, xml, xmlreader, xmlrpc, xmlwriter, xsl, zip, cgi-fcgi, imagick, apcu, ionCube Loader, Zend OPcache

**Nextcloud version:** 18.0.1 - 18.0.1.3

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
(
)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.4.0
 - activity: 2.11.0
 - admin_audit: 1.8.0
 - apporder: 0.9.0
 - bookmarks: 2.3.4
 - bruteforcesettings: 1.5.0
 - calendar: 2.0.1
 - cloud_federation_api: 1.1.0
 - comments: 1.8.0
 - contacts: 3.1.9
 - dav: 1.14.0
 - federatedfilesharing: 1.8.0
 - federation: 1.8.0
 - files: 1.13.1
 - files_pdfviewer: 1.7.0
 - files_rightclick: 0.15.2
 - files_sharing: 1.10.1
 - files_trashbin: 1.8.0
 - files_versions: 1.11.0
 - firstrunwizard: 2.7.0
 - groupfolders: 6.0.1
 - issuetemplate: 0.6.0
 - logreader: 2.3.0
 - lookup_server_connector: 1.6.0
 - maps: 0.1.5
 - nextcloud_announcements: 1.7.0
 - notes: 3.1.5
 - notifications: 2.6.0
 - oauth2: 1.6.0
 - password_policy: 1.8.0
 - photos: 1.0.0
 - privacy: 1.2.0
 - provisioning_api: 1.8.0
 - quota_warning: 1.7.0
 - recommendations: 0.6.0
 - serverinfo: 1.8.0
 - settings: 1.0.0
 - sharebymail: 1.8.0
 - support: 1.1.0
 - systemtags: 1.8.0
 - tasks: 0.11.3
 - text: 2.0.0
 - theming: 1.9.0
 - twofactor_backupcodes: 1.7.0
 - updatenotification: 1.8.0
 - viewer: 1.2.0
 - workflowengine: 2.0.0
Disabled:
 - deck
 - encryption
 - files_external
 - files_videoplayer
 - onlyoffice
 - survey_client
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "nextcloud.wallaustrasse.de",
        "owncloud.wallaustrasse.de"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "overwrite.cli.url": "https:\/\/nextcloud.wallaustrasse.de",
    "overwriteprotocol": "https",
    "forcessl": true,
    "forceSSLforSubdomains": true,
    "dbtype": "mysql",
    "version": "18.0.1.3",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbtableprefix": "oc_",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "logtimezone": "UTC",
    "installed": true,
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpmode": "smtp",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "loglevel": 1,
    "theme": "",
    "maintenance": false,
    "trashbin_retention_obligation": "auto",
    "updater.release.channel": "stable",
    "mail_smtpsecure": "ssl",
    "mail_smtpauth": 1,
    "mail_smtpauthtype": "LOGIN",
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "465",
    "mail_sendmailmode": "smtp",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "app_install_overwrite": [
        "calendar"
    ],
    "mysql.utf8mb4": true
}
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** 

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

## Client configuration

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:73.0) Gecko/20100101 Firefox/73.0

**Operating system:** Windows 10

## Logs

<details><summary>Web server error log</summary>

```
Insert your web server log here 
```
</details>

<details><summary>Nextcloud log</summary>

```
no new log entries were generated when trying to import the calendar
```
</details>

<details><summary>Browser log</summary>

Insert your browser log here, this could for example include:

	a) The javascript console log
	b) The network log
	c) ...
</details>

**Comment by georgehrke** [comment]
@sreutter This is most likely a duplicate of https://github.com/nextcloud/calendar/issues/1898. Can you please check https://github.com/nextcloud/calendar/issues/1898 and see if the described bug is the same as yours?

**Comment by sreutter** [comment]
yes, looks like it's the same problem. there is no "import" window too.

**Comment by sreutter** [comment]
I tested the file with this ics validator:
https://icalendar.org/validator.html#results
it says "Success! No errors found."

**Comment by sreutter** [comment]
Tried it with Edge, same result. 
In Chrome, it works!

here the complete ics file:

BEGIN:VCALENDAR
PRODID:-//Grafik-Partner GmbH//Muellmax 7.0 MIMEDIR//EN
VERSION:2.0
METHOD:PUBLISH
BEGIN:VEVENT
DTSTART;VALUE=DATE:20200227
DTEND;VALUE=DATE:20200228
DESCRIPTION:Entsorgungsbetrieb der Stadt Mainz\n
	Zwerchallee 24\n
	55120 Mainz\n
	\n
	Telefon: +49 6131 12 34 56 (Abfallberatung)\n
	Telefax: +49 6131 12 34 83\n
	\n
	Der Entsorgungsbetrieb ist ein Eigenbetrieb der Stadt Mainz.
LOCATION:55118 Mainz-Neustadt, Wallaustraße 31
RRULE:FREQ=WEEKLY;UNTIL=20210103T000000Z;
 INTERVAL=1;BYDAY=TH;WKST=MO
RDATE;VALUE=DATE:
 20200228,
 20200408,
 20200417,
 20200429,
 20200522,
 20200605,
 20200612,
 20201222,
 20201230
EXDATE;VALUE=DATE:
 20200227,
 20200409,
 20200416,
 20200430,
 20200521,
 20200604,
 20200611,
 20201224,
 20201231
TRANSP:TRANSPARENT
SEQUENCE:0
UID:15828015991@muellmax.de
DTSTAMP:20200227T110639Z
SUMMARY:EB Restabfall-Behälter
CLASS:PUBLIC
CREATED:20200225T161546Z
LAST-MODIFIED:20200225T161546Z
URL:http://www.muellmax.de
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20200305
DTEND;VALUE=DATE:20200306
DESCRIPTION:Entsorgungsbetrieb der Stadt Mainz\n
	Zwerchallee 24\n
	55120 Mainz\n
	\n
	Telefon: +49 6131 12 34 56 (Abfallberatung)\n
	Telefax: +49 6131 12 34 83\n
	\n
	Der Entsorgungsbetrieb ist ein Eigenbetrieb der Stadt Mainz.
LOCATION:55118 Mainz-Neustadt, Wallaustraße 31
RRULE:FREQ=WEEKLY;UNTIL=20201227T000000Z;
 INTERVAL=2;BYDAY=TH;WKST=MO
RDATE;VALUE=DATE:
 20200417,
 20200429,
 20200612,
 20201222
EXDATE;VALUE=DATE:
 20200416,
 20200430,
 20200611,
 20201224
TRANSP:TRANSPARENT
SEQUENCE:0
UID:15828015992@muellmax.de
DTSTAMP:20200227T110639Z
SUMMARY:EB Papier-Behälter
CLASS:PUBLIC
CREATED:20200225T161546Z
LAST-MODIFIED:20200225T161546Z
URL:http://www.muellmax.de
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20200227
DTEND;VALUE=DATE:20200228
DESCRIPTION:Entsorgungsbetrieb der Stadt Mainz\n
	Zwerchallee 24\n
	55120 Mainz\n
	\n
	Telefon: +49 6131 12 34 56 (Abfallberatung)\n
	Telefax: +49 6131 12 34 83\n
	\n
	Der Entsorgungsbetrieb ist ein Eigenbetrieb der Stadt Mainz.
LOCATION:55118 Mainz-Neustadt, Wallaustraße 31
RRULE:FREQ=WEEKLY;UNTIL=20210103T000000Z;
 INTERVAL=4;BYDAY=TH;WKST=MO
RDATE;VALUE=DATE:
 20200228,
 20200522
EXDATE;VALUE=DATE:
 20200227,
 20200521
TRANSP:TRANSPARENT
SEQUENCE:0
UID:15828015993@muellmax.de
DTSTAMP:20200227T110639Z
SUMMARY:EB Glas-Tonne
CLASS:PUBLIC
CREATED:20200225T161546Z
LAST-MODIFIED:20200225T161546Z
URL:http://www.muellmax.de
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20200227
DTEND;VALUE=DATE:20200228
DESCRIPTION:Entsorgungsbetrieb der Stadt Mainz\n
	Zwerchallee 24\n
	55120 Mainz\n
	\n
	Telefon: +49 6131 12 34 56 (Abfallberatung)\n
	Telefax: +49 6131 12 34 83\n
	\n
	Der Entsorgungsbetrieb ist ein Eigenbetrieb der Stadt Mainz.
LOCATION:55118 Mainz-Neustadt, Wallaustraße 31
RRULE:FREQ=WEEKLY;UNTIL=20210103T000000Z;
 INTERVAL=1;BYDAY=TH;WKST=MO
RDATE;VALUE=DATE:
 20200228,
 20200408,
 20200417,
 20200429,
 20200522,
 20200605,
 20200612,
 20201222,
 20201230
EXDATE;VALUE=DATE:
 20200227,
 20200409,
 20200416,
 20200430,
 20200521,
 20200604,
 20200611,
 20201224,
 20201231
TRANSP:TRANSPARENT
SEQUENCE:0
UID:15828015994@muellmax.de
DTSTAMP:20200227T110639Z
SUMMARY:EB Bio-Tonne
CLASS:PUBLIC
CREATED:20200225T161546Z
LAST-MODIFIED:20200225T161546Z
URL:http://www.muellmax.de
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20200310
DTEND;VALUE=DATE:20200311
DESCRIPTION:Entsorgungsbetrieb der Stadt Mainz\n
	Zwerchallee 24\n
	55120 Mainz\n
	\n
	Telefon: +49 6131 12 34 56 (Abfallberatung)\n
	Telefax: +49 6131 12 34 83\n
	\n
	Der Entsorgungsbetrieb ist ein Eigenbetrieb der Stadt Mainz.
LOCATION:55118 Mainz-Neustadt, Wallaustraße 31
RRULE:FREQ=WEEKLY;UNTIL=20210103T000000Z;
 INTERVAL=2;BYDAY=TU;WKST=MO
RDATE;VALUE=DATE:
 20200603
EXDATE;VALUE=DATE:
 20200602
TRANSP:TRANSPARENT
SEQUENCE:0
UID:15828015995@muellmax.de
DTSTAMP:20200227T110639Z
SUMMARY:EB Gelber Sack
CLASS:PUBLIC
CREATED:20200225T161546Z
LAST-MODIFIED:20200225T161546Z
URL:http://www.muellmax.de
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20200421
DTEND;VALUE=DATE:20200422
DESCRIPTION:Entsorgungsbetrieb der Stadt Mainz\n
	Zwerchallee 24\n
	55120 Mainz\n
	\n
	Telefon: +49 6131 12 34 56 (Abfallberatung)\n
	Telefax: +49 6131 12 34 83\n
	\n
	Der Entsorgungsbetrieb ist ein Eigenbetrieb der Stadt Mainz.
LOCATION:55118 Mainz-Neustadt, Wallaustraße 31
RRULE:FREQ=WEEKLY;UNTIL=20201206T000000Z;
 INTERVAL=8;BYDAY=TU;WKST=MO
TRANSP:TRANSPARENT
SEQUENCE:0
UID:15828015996@muellmax.de
DTSTAMP:20200227T110639Z
SUMMARY:EB Elektroschrott-Tonne
CLASS:PUBLIC
CREATED:20200225T161546Z
LAST-MODIFIED:20200225T161546Z
URL:http://www.muellmax.de
END:VEVENT
END:VCALENDAR

**Comment by georgehrke** [comment]
Okay thx!

Closing as Duplicate of #1898 in that case.
Issue will be fixed with Calendar 2.0.2
